src/Normalizer/UriNormalizer.php

```diff
@@ -58,11 +49,13 @@ public function denormalize($data, $class, $format = null, array $context = [])
         return $object;
     }
 
-    /**
-     * {@inheritdoc}
-     */
     public function supportsDenormalization($data, $type, $format = null)
     {
         return isset(self::$supportedTypes[$type]);
     }
+
+    public function getSupportedTypes(?string $format): array
+    {
+        return self::$supportedTypes;
+    }
```

Comment:
Is this a new part of the interfaces?

Reply:
Yes, this is now a required method of the normalizers. 👍